Inline (tab-style) navigation

TrebBrennan opened the pull request:
## Definition of Done
- [x] Content/documentation reviewed by Julian or someone in the Content Team
- [x] UX reviewed by Gary or someone the Design team
- [x] Code reviewed by one of the core developers
- [x] Acceptance Testing
  - [x] HTML5 validation (CircleCI)
- [x] Stakeholder/PO review
- [x] CHANGELOG updated

TrebBrennan committed:
Start of inline nav feature

Added a helper class (controversial!) I’m aware we want to avoid
Classitis, but `.reader` is useful as it is something which should to
be obvious in HTML and can also be extended

TrebBrennan committed:
Mobile styles and change to reader class name

TrebBrennan committed:
Mobile styles and change to reader class name

TrebBrennan committed:
Classes match to branch #205

klepas commented on assets/sass/_helper.scss:
```diff
@@ -0,0 +1,17 @@
+
+// Added as a class because this should to be obvious in HTML
+
+.is-visuallyhidden {
```
Just realised: this could be a nice opportunity to start a documentation section called _Accessibility_, and document this there… :D

@petronbot, @TrebBrennan what do you reckon?

Re. names, should the partial be `_helpers.scss` or would `_accessibility.scss` be apt?

(Noting we don’t have to name the partial after the documentation section.)

TrebBrennan replied:
Yeah makes sense. @jonathanconway's PR #213 has that naming in it too :)
Good pickup

klepas commented:
Added a note re. the visually hidden a11y helper class buried in this PR. :)

klepas commented:
@TrebBrennan might be a good point to get @joolswood involved?

Commit:
Changelog ammendment.

Commit:
Moves .visuallyhidden into accessibility partial.

Commit:
Further mergin'.

Commit:
Amends wrong classname for inline tab navs.

TrebBrennan committed:
Update from develop

TrebBrennan committed:
Merge branch 'develop' into feature/DESIGN-147_inline-nav

TrebBrennan committed:
Merge branch 'develop' into feature/DESIGN-147_inline-nav

TrebBrennan committed:
Documentation and class update

Commit:
Improve for scanning

The `(experimental)` might be a bit too much, especially as we don’t
use it elsewhere.

See what you think @TrebBrennan

Commit:
Quick fix

hannah-ustwo commented:
STATUS UPDATE: This PR was put on hold/deprioritised as it was not included in the demo release on 4 Aug. It is now scheduled to be addressed (finally!)  in the current sprint 32.

TrebBrennan committed:
Merge branch 'develop' into feature/DESIGN-147_inline-nav

# Conflicts:
#	assets/sass/_navigation.scss

TrebBrennan committed:
previous conflict commit error

TrebBrennan committed:
Merge branch 'develop' into feature/DESIGN-147_inline-nav

maxious commented on CHANGELOG.md:
```diff
@@ -130,10 +135,28 @@ Build environment:
 Adds or modifies:
 
+<<<<<<< HEAD
+<<<<<<< e0da93c6e08e5ffa6c1351f1ab5d97ee2d47b40d
```
Lotta git conflict junk here

maxious commented:
As stakeholder for platforms SCSS rationalisation, LGTM.

TrebBrennan commented:
Note, @maxious has kindly offered to clean up the changelog mess.

maxious committed:
Fix v1.3.0 Changelog
